misc/balena-etcher-bin/douninst.sh

```diff
@@ -0,0 +1 @@
+rm -rf /opt/balena-etcher
```

Comment:
i don't think this is the correct way to remove a directory
it should be removed via removepkg, not douninst.sh